### PR TITLE
Add JSON, MsgPack and Cbor Serializers 

### PR DESCRIPTION
**Pull request by rubyonrails3**

### CODE REVIEW

File: lib/wampproto/serializer/json.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "json"
+
+module Wampproto
+  module Serializer
+    # Add common API for serializer
+    class JSON
+      def self.encode(message)
```
Comment (om26er) [inline lib/wampproto/serializer/json.rb:9]:
```suggestion
      def self.serialize(message)
```

File: lib/wampproto/serializer/json.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        ::JSON.dump(message)
+      end
+
+      def self.decode(message)
```
Comment (om26er) [inline lib/wampproto/serializer/json.rb:13]:
```suggestion
      def self.deserialize(message)
```

Comment (rubyonrails3) [reply lib/wampproto/serializer/json.rb]: Updated the code @om26er

File: lib/wampproto/serializer/json.rb (hunk 1 above)
Comment (rubyonrails3) [reply lib/wampproto/serializer/json.rb]: @om26er  updated the PR

**Review by om26er — APPROVED** [review]

**Pull request merged**